yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -215,6 +215,10 @@ func updateImportedEventsCountsInTheRow(row *rowData, tableName string, schemaNa
 		return fmt.Errorf("data file descriptor not found at %s for snapshot", dataFileDescriptorPath)
 	}
 
+	if schemaName != "public" && schemaName != "" { //multiple schema specific 
```

Comment:
You should also add the case "if source db type is PG"  I guess? This will break for cases of non-pg where schemaName is not public, right?

Reply:
Yes, correct. I wanted to get this checked and thought this would be checked in the automation of oracle live but I just realised we are not using non-public schema in those cases. Will fix one test case in a separate PR. Now we need it as we are dealing with schema at a lot of places with PG support.